website_rma/views/res_config_settings_views.xml

```diff
@@ -16,7 +16,6 @@
                         title="Set the default RMA Team and RMA responsible of the RMAs created from the Request RMA page."
                     >
                         <div class="o_setting_right_pane">
-                            <label for="id" string="Request RMA Form" />
                             <div
                                 class="text-muted"
                                 id="msg_contact_form_install_setting"
```

Comment:
Looks like this id slipped from copy/paste:
```suggestion
                                id="website_rma_settings"
```

Reply:
Great, going to update the PR